test-container/src/main/java/io/strimzi/StrimziKafkaContainer.java

```diff
@@ -73,18 +76,24 @@
         LOGGER.info("Supported Strimzi version: {}", STRIMZI_VERSION);
     }
 
-    public StrimziKafkaContainer(final String version) {
+    public StrimziKafkaContainer(final String version, Map<String, String> additionalKafkaConfiguration) {
```

Comment:
Perhaps rename `version` to `imageVersion`, since that's what it seems to be (rather than a Kafka version string).